docs/config.md

```diff
@@ -328,6 +328,8 @@ Thrift servers define additional parameters:
    currently supports 'binary' for `TBinaryProtocol` (default) and
    'compact' for `TCompactProtocol`. Typically this setting matches
    the router's client `thriftProtocol` param.
+* *attemptTTwitterUpgrade* -- controls whether thrift protocol upgrade should be
+   attempted.  (default: true)
```

Comment:
I think this actually belongs in the *client* section of this doc, starting on line 336 below.